src/main/java/org/apache/commons/imaging/ImageParser.java

```diff
@@ -106,7 +98,7 @@
      *
      * @return A valid array of image parsers
      */
-    public static ImageParser[] getAllImageParsers() {
+    public static ImageParser<?>[] getAllImageParsers() {
```

Comment:
Should this return a list for better compatibility with generics?

Reply:
Good idea, done!